### ISSUE TITLE
[Chips] Support focus ring when it's focused

### ISSUE THREAD

**Issue by PinkyJie** [material-components/material-components-android#4448]
**Is your feature request related to a problem? Please describe.**
When the chip is focused by keyboard (with tab), it doesn't show the focus ring. According to the spec described in Material Design website, there should be a 3dp focus ring on the chip when it's focused.

![image](https://github.com/user-attachments/assets/57e71c1d-5531-4396-8f8a-b6c2d4d0bce8)

**Comment by hunterstich** [comment]
Hey @PinkyJie. Thanks for filing an issue.

This is something design would like and we've investigated. Updating the way focus is shown would be something we'd want to be consistent across all Android Views - not just Material Components. Ultimately this would be something we need to collaborate with on the Android Framework and is a larger work stream than what we usually track here. 

I believe the Material Design website has kept these tokens because other platforms, like web, don't have the same limitations and are able to use them. 

Closing for now but feel free to continue the conversation.
